content/integrations/new_relic.md

```diff
@@ -20,22 +20,25 @@ Connect to New Relic to:
 ### New Relic Alerts in Event Stream
 
 1.  On the Webhook tab of New Relic's alerting notification settings page, enter the following webhook URL:
+`https://app.datadoghq.com/intake/webhooknewrelic?api_key={YOUR_DATADOG_API_KEY}`
 
-        https://app.datadoghq.com/intake/webhook/newrelic?api_key={YOUR_DATADOG_API_KEY}
-
-1.  For 'Custom Payload'(s), select JSON 'Payload Type'.
+2.  For 'Custom Payload'(s), select JSON 'Payload Type'.
 
 ### New Relic APM Metric Collection
 
 1.  Locate your API key on New Relic's API Keys page (**Account Settings** -> **Integrations** -> **API Keys**) and enter it in the form on the [Datadog New Relic Integration](https://app.datadoghq.com/account/settings#integrations/new_relic) page.
+  *  *Metrics can only be imported for New Relic customers at the Pro level or above.*
+  * *Make sure you're using the the REST API Key, the User API Keys won't work*
```

Comment:
As my other comment says, we can omit this if we just say in step 1, "Locate your **REST API key**..."